### ISSUE TITLE
IDE/code editor props autocomplete

### ISSUE THREAD

**Issue by Seminioni** [morphatic/vuetify-extension-template#2]
Hi @morphatic. 
Do you have an IDE/code editor autocomplete for props in templates' section if you're extending a vuetify component like you described?

**Comment by morphatic** [comment]
Interesting question. You're talking about one that would be specific to your extension? If you build it with TypeScript, I imagine that a TS-aware editor might be able to infer props from your type definitions, but I did not include anything more generic in this template

**Comment by Seminioni** [comment]
No, I'm talking about those props that are extended by our component.

Look what I mean. Here I define `Divider` component that extends `VDivider`.
<img width="657" alt="изображение" src="https://user-images.githubusercontent.com/16149737/99957448-afb96400-2dc2-11eb-9ebb-c7b4670bd462.png">

In the next component I'm using `Divider` like always. But the issue is that I don't have full autocomplete enabled after extending. The only visible prop is `type`.
<img width="747" alt="изображение" src="https://user-images.githubusercontent.com/16149737/99957631-fa3ae080-2dc2-11eb-83a5-18909ff8e77d.png">
